client/css/style.css

```diff
@@ -915,6 +915,13 @@ button, .user {
 	padding-top: 2px;
 	line-height: 1.8;
 }
+#settings #change-password .error {
+	color: #e74c3c;
+	margin-bottom: 1em;
+}
+#settings #change-password .error.success {
+	color: #2ecc40;
```

Comment:
Yes, it seems very messy to have elements that are both `.error` and `.success`. You should use one or the other, and if you do need a common class, you could call it `.feedback`. If it's just for common CSS properties though, `.error, .success { /* ... */ }` will do just fine.